From 8f0d7490fa7ab3e20f7837c264c66eade2502223 Mon Sep 17 00:00:00 2001
From: Eric Anholt <eric@anholt.net>
Date: Mon, 25 Mar 2019 16:30:18 -0700
Subject: [PATCH 19/29] glamor: Fix a compiler warning since the recent OOM
 fixes.

Signed-off-by: Eric Anholt <eric@anholt.net>
(cherry picked from commit 3b26b90cb787a14fa5f8bb2033eab8ab6562a9a5)
---
 glamor/glamor_prepare.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/glamor/glamor_prepare.c b/glamor/glamor_prepare.c
index c1a611f9a..835c4ebea 100644
--- a/glamor/glamor_prepare.c
+++ b/glamor/glamor_prepare.c
@@ -146,8 +146,6 @@ glamor_prep_pixmap_box(PixmapPtr pixmap, glamor_access_t access, BoxPtr box)
 static void
 glamor_fini_pixmap(PixmapPtr pixmap)
 {
-    ScreenPtr                   screen = pixmap->drawable.pScreen;
-    glamor_screen_private       *glamor_priv = glamor_get_screen_private(screen);
     glamor_pixmap_private       *priv = glamor_get_pixmap_private(pixmap);
 
     if (!GLAMOR_PIXMAP_PRIV_HAS_FBO(priv))
-- 
2.11.0

